CHANGELOG.rst

```diff
@@ -2,11 +2,25 @@ Changelog
 =========
 
 
-2.0.0-pre.rc.6-lts
+2.0.0-pre.rc.7
 ---------
 
 Features
 ~~~~~~~~
+- 2338 Add `cargo-all-features` instrumentation.
```

Comment:
Please consider prefixing a number with `hyperledger#`. It enables linking to issues and PRs

![image](https://user-images.githubusercontent.com/49983831/182821921-f53e0fa2-b56b-4c2a-9989-5f968eea327d.png)

Reply:
Done that. We'll nee to be a bit careful with older changelog entries.